File: examples/OmeroClients/rcollection.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
from omero.rtypes import rstring, rint, rset, rlist

# Sets and Lists may be interpreted differently on the server
list = rlist(rstring("a"), rstring("b"))
set = rset(rint(1), rint(2))

File: examples/OmeroClients/rcollection.java


public class rcollection {
    public static void main(String[] args) {
        // Sets and Lists may be interpreted differently on the server
        omero.RList list = omero.rtypes.rlist(omero.rtypes.rstring("a"), omero.rtypes.rstring("b"));
        omero.RSet set = omero.rtypes.rset(rint(1), rint(2));
    }
}
